lib/utils.js

```diff
@@ -111,6 +112,7 @@ exports.fuse = function fuse (target, handlers) {
 // merges a object's proppertys / values with a other object
 exports.merge = function merge (target, obj) {
   for (var i in obj) {
```

Comment:
I've become a big fan of Object.keys over the years as well. It's pretty fast these days and you'll be iterating over an array.